pkgs/applications/video/recordmydesktop/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchsvn, automake, autoconf, zlib, popt, xorg, libvorbis, libtheora }:
+
+stdenv.mkDerivation rec {
+  name = "recordmydesktop-${version}";
+  version = "0.3.8-svn602";
+
+  src = fetchsvn {
```

Comment:
According to http://sourceforge.net/p/recordmydesktop/svn/602/log/?path=/trunk/recordmydesktop/ChangeLog `0.3.8.1` is revision `r584`. I did the same as Debian https://packages.debian.org/sid/recordmydesktop. I could have named version `0.3.8.1-svn602` instead of `0.3.8-svn602` though.